test/membrane_flv_plugin/demuxer_test.exs

```diff
@@ -10,7 +10,10 @@ defmodule Membrane.FLV.Demuxer.Test do
     def handle_init(input_file_path) do
       spec = %ParentSpec{
         children: [
-          src: %Membrane.File.Source{location: input_file_path},
+          src: %Membrane.File.Source{
+            location: input_file_path,
+            content_format: Membrane.FLV
```

Comment:
Why exactly do we need to do this? We could just accept unknown content type in demuxer, right?